### ISSUE TITLE
Error converting VTK to VTU

### ISSUE THREAD

**Issue by keurfonluu** [nschloe/meshio#831]
**Platform**: Windows 10 (1909)
**Python**: 3.7.7
**meshio**: 4.0.14
**numpy**: 1.18.1

When reading a VTK file (written by `meshio`) and converting it to VTU, it raises a `KeyError: dtype('>i4')` as the data type of cells connectivity is `dtype('>i4')` and not `dtype('int32')`.

Code to reproduce the error:

```python
import meshio
import numpy

points = numpy.array([[0.0, 0.0, 0.0], [1.0, 0.0, 0.0], [0.0, 1.0, 0.0]])
cells = [("triangle", numpy.array([[0, 1, 2]]))]
meshio.write_points_cells("test.vtk", points, cells)
mesh = meshio.read("test.vtk")
print(mesh.cells[0].data.dtype)  # dtype('>i4')
mesh.write("test.vtu")  # KeyError: dtype('>i4')
```

**Comment by nschloe** [comment]
Ah, big-endian. There are some bugs for this architecture-type. I'd love to get my hands ona be machine for debugging.

**Comment by nschloe** [comment]
Fixed by #832.
